README.md

```diff
@@ -24,6 +24,7 @@ Ansible role for a bigbluebutton installation (following the documentation on ht
 | `bbb_greenlight_secret` | Secret for greenlight _(required when using greenlight)_ |  | can be generated with `openssl rand -hex 64`
 | `bbb_greenlight_db_password` | Password for greenlight's database  _(required when using greenlight)_ | | can be generated with `openssl rand -hex 16`
 | `bbb_greenlight_default_registration` | Registration option open(default), invite or approval
+| `bbb_allow_mail_notifications`  | Require E-Mail verification for greenlight account creation | `true` |
```

Comment:
Could you maybe also change the description to be similar to the one provided in the .env?

Reply:
:+1: i will do that